src/database/models/role.js

```diff
@@ -0,0 +1,27 @@
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class Role extends Model {
+    /**
+     * Helper method for defining associations.
```

Comment:
These comments cause linting errors and they don't really do anything or guide anyone. Please remove them

Reply:
@Procech20 please learn what JSDOCs are and what they do for JS